flink-connector-obkv-hbase/src/test/java/com/oceanbase/connector/flink/OBKVHBaseConnectorByODPITCase.java

```diff
@@ -0,0 +1,361 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oceanbase.connector.flink;
+
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.table.api.EnvironmentSettings;
+import org.apache.flink.table.api.bridge.java.StreamTableEnvironment;
+
+import org.apache.hadoop.hbase.util.Bytes;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.lifecycle.Startables;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.testcontainers.shaded.org.awaitility.Awaitility.given;
+
+public class OBKVHBaseConnectorByODPITCase extends OceanBaseMySQLTestBase {
+    private static final Logger LOG = LoggerFactory.getLogger(OBKVHBaseConnectorByODPITCase.class);
+    private static final String CLUSTER_NAME = "flink-oceanbase-ci";
+    private static final String APP_NAME = "odp_test";
+    private static final int ODP_PORT = 2883;
+    private static final int RPC_PORT = 2885;
+    private static final String ODP_USERNAME = "root@sys";
+    private static final String PROXY_USERNAME = "proxyro@sys";
+    private static final String ODP_PASSWORD = "123456";
+    private GenericContainer<?> obproxy;
+    private static final Pattern COMMENT_PATTERN = Pattern.compile("^(.*)--.*$");
+
+    public Map<String, Object> getODPOptions() {
+        Map<String, Object> options = new HashMap<>();
+        options.put("url", getODPUrl());
+        options.put("odp-mode", true);
+        options.put("username", ODP_USERNAME);
+        options.put("password", ODP_PASSWORD);
+        options.put("sys.username", getUsername());
```

Comment:
It's wired to use test username here, is `sys.username` really required by the client?